ts/addon.ts

```diff
@@ -17,8 +17,8 @@ export default addon({
   included() {
     this._super.included.apply(this, arguments);
     this._checkDevelopment();
-    this._checkPeerVersions();
-    this._checkAddonAppFiles();
+    this._checkAddonAppFiles();y
```

Comment:
You've an extra `y` here, @dfreeman!

Reply:
…looks like you fixed it. 😆

Reply:
Yep, that was the result of resolving a merge conflict on my phone 🙃